### DISCUSSION TITLE
[Feature] Provide official cloud services

### DISCUSSION THREAD

**Discussion by weironz** [immich-app/immich#14536] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Sometimes I don't want to set up my own server due to the complexity of personal maintenance and data security issues. Will there be an official IMMICH cloud service that I can subscribe to?

I have been using Google Photos, but Google ONE restricts usage in some areas, such as China, and when the capacity exceeds the free limit, there is nothing I can do.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
We are aiming to do something like this at some point yes. Since this is not a feature request for the Immich software per se though, I'll close this thread.
